### DISCUSSION TITLE
后续会加战斗的每日任务和自动地脉花吗？

### DISCUSSION THREAD

**Discussion by ttezy** [babalae/better-genshin-impact#689] [General]
首先非常感谢各位开发者制作的工具，感谢你们的付出！在这里想问一下未来是否有计划加入自动完成每日委托和自动清理地脉花的功能？委托方面目前可以使用的应该是自动战斗，但是每一个委托都需要手动进入到委托位置然后开启自动战斗功能、战斗结束后再手动关闭才可以完成，实际使用起来还是需要人坐在电脑前看着屏幕，没有办法像自动秘境一样完全离开。另一方面就是刷地脉花也是一样的情况。200体力刷完要手动找五个地脉花并且手动开关自动战斗，要一直盯着屏幕看才可以。

我经常使用自动采集的脚本感觉也很好用，不知道地脉花可否使用相似的思路实现。只用对一个国家的地脉花坐标进行标记然后自动导航到位置上后开启自动战斗。

委托的话也是选择最简单的蒙德只完成战斗委托就可以。

不知道有没有可以简单实现这两个功能的方法或脚本，欢迎大家讨论。

**Comment by TauDong** [comment]
跟采集不太一样，地脉花是每天的位置和产物不同的，不能简单的用坐标导航，需要先在地图上识别出当天该地区的地脉位置，打完一个之后下一个地脉花的位置就是按固定路线循环的了
